src/DataProtection/DataProtection/test/AuthenticatedEncryption/ConfigurationModel/CngGcmAuthenticatedEncryptorDescriptorTests.cs

```diff
@@ -39,7 +40,7 @@ public void ExportToXml_WithProviders_ProducesCorrectPayload()
         public void ExportToXml_WithoutProviders_ProducesCorrectPayload()
         {
             // Arrange
-            var masterKey = "k88VrwGLINfVAqzlAp7U4EAjdlmUG17c756McQGdjHU8Ajkfc/A3YOKdqlMcF6dXaIxATED+g2f62wkRRRRRzA==".ToSecret();
+            var masterKey = Convert.ToBase64String(Encoding.UTF8.GetBytes("[PLACEHOLDER]"));
```

Comment:
Also need `.ToSecret()` on [line 48](https://github.com/dotnet/aspnetcore/pull/32659/files#diff-13752fe59703493a72812ffd84354ea0be138d2567263ac94a314ef44d6db624R48) because you're changing `masterKey`'s type.
``` text
/Users/runner/work/1/s/src/DataProtection/DataProtection/test/AuthenticatedEncryption/ConfigurationModel/CngGcmAuthenticatedEncryptorDescriptorTests.cs(48,16): error CS1503: Argument 2: cannot convert from 'string' to 'Microsoft.AspNetCore.DataProtection.ISecret'
```

Reply:
Whoops, good catch